erizo_controller/erizoClient/src/Room.js

```diff
@@ -83,6 +83,7 @@ const Room = (altIo, altConnectionHelpers, altConnectionManager, specInput) => {
     if (!that.p2p) {
       stream.pc.addStream(stream);
     }
+    stream.state = 'subscribed';
```

Comment:
In the future we should think about having a more involved fsm like we're using in other places. Even maybe just having all the states set as a constant somewhere so we can see all the possible states at a glance would be an improvement.